pyproject.toml

```diff
@@ -87,3 +85,10 @@ exclude = '''
 packages = ["rst2pdf"]
 
 [tool.setuptools_scm]
+
+[tool.uv]
+dev-dependencies = [
```

Comment:
[uv shipped dependency-group support in 0.4.27](https://github.com/astral-sh/uv/releases/tag/0.4.27), since that's a standardised way to do groups it might make sense to use that instead of this uv-specific version?

Reply:
Thanks for this tip! I adapted the structure to add a dev dependency group.